chrismeeky/PropertyPro
# #167006726 users can update property data endpoint

#### What does this PR do?
Create endpoints for user posting new property
#### Description of Task to be completed?
Have the following endpoint working:
POST /api/v1/property/:id
#### How should this be manually tested?
after cloning to local repo, use postman to test the enpoint with the following setup:
Under the Post Body, select x-www-urlencoded
Complete the fields and make sure the method is Set to POST
#### What are the relevant pivotal tracker stories?
#167006726
